plugins/CoreAdminHome/OptOutManager.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\CoreAdminHome;
+
+use Piwik\Common;
+use Piwik\Nonce;
+use Piwik\Plugins\LanguagesManager\API as APILanguagesManager;
+use Piwik\Plugins\LanguagesManager\LanguagesManager;
+use Piwik\Plugins\PrivacyManager\DoNotTrackHeaderChecker;
+use Piwik\Tracker\IgnoreCookie;
+use Piwik\Url;
+use Piwik\View;
+
+class OptOutManager
+{
+    /** @var DoNotTrackHeaderChecker */
+    private $doNotTrackHeaderChecker;
+
+    /** @var array */
+    private $javascripts;
+
+    /** @var array */
+    private $stylesheets;
+
+    /** @var string */
+    private $title;
+
+    /**
+     * @param DoNotTrackHeaderChecker $doNotTrackHeaderChecker
+     */
+    public function __construct(DoNotTrackHeaderChecker $doNotTrackHeaderChecker = null)
+    {
+        $this->doNotTrackHeaderChecker = $doNotTrackHeaderChecker ?: new DoNotTrackHeaderChecker();
+
+        $this->javascripts = array(
+            'inline' => array(),
+            'external' => array(),
+        );
+
+        $this->stylesheets = array(
+            'inline' => array(),
+            'external' => array(),
+        );
+    }
+
+    /**
+     * @param string $javascript
+     * @param bool $inline
+     */
+    public function addJavascript($javascript, $inline = true)
+    {
+        $type = $inline ? 'inline' : 'external';
+        $this->javascripts[$type][] = $javascript;
+    }
+
+    /**
+     * @return array
+     */
+    public function getJavascripts()
+    {
+        return $this->javascripts;
+    }
+
+    /**
+     * @param string $stylesheet
+     * @param bool $inline
+     */
+    public function addStylesheet($stylesheet, $inline = true)
+    {
+        $type = $inline ? 'inline' : 'external';
+        $this->stylesheets[$type][] = $stylesheet;
+    }
+
+    /**
+     * @return array
+     */
+    public function getStylesheets()
+    {
+        return $this->stylesheets;
+    }
+
+    /**
+     * @return string
+     */
+    public function getTitle()
+    {
+        return $this->title;
+    }
+
+    /**
+     * @param string $title
+     */
+    public function setTitle($title)
+    {
+        $this->title = $title;
+    }
+
+    /**
+     * @return View
+     * @throws \Exception
+     */
+    public function createView()
```

Comment:
Just wondering if the code of this method could be as well in the controller? Or is it supposed to be here so one can override it? Do you need this feature?